fastinfoset/src/main/java/com/sun/xml/fastinfoset/Decoder.java

```diff
@@ -1938,28 +1954,20 @@ protected final boolean _isFastInfosetDocument() throws IOException {
                 _octetBuffer[3] != EncodingConstants.BINARY_HEADER[3]) {
             
             // Check for each form of XML declaration
-            for (int i = 0; i < EncodingConstants.XML_DECLARATION_VALUES.length; i++) {
-                _octetBufferLength = EncodingConstants.XML_DECLARATION_VALUES[i].length - _octetBufferOffset;
+            for (byte[] XML_DECLARATION_VALUES : EncodingConstants.XML_DECLARATION_VALUES) {
```

Comment:
use lower case for non-static variable name

Reply:
Yes, looks like this was caused by automatic Netbeans changes.